acceptance/tests/cli/scenario_test.go

```diff
@@ -0,0 +1,38 @@
+package cli
```

Comment:
[nit] would naming this file `upgrade_test.go` be more descriptive?

Reply:
Yes, I thought the same thing. Now that you mention it too, I think it's the right thing.